src/Microsoft.Framework.HashCodeCombiner.Sources/Microsoft.Framework.HashCodeCombiner.Sources.xproj

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">14.0</VisualStudioVersion>
+    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
+  </PropertyGroup>
+  <Import Project="$(VSToolsPath)\DNX\Microsoft.DNX.Props" Condition="'$(VSToolsPath)' != ''" />
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>31d4c34c-436f-48e9-b8e4-641574741887</ProjectGuid>
+    <RootNamespace>Microsoft.Framework.HashCodeCombiner.Sources</RootNamespace>
```

Comment:
Where is this script? I don't see any scripts in the root of this repo?

(For the record, it does affect the default namespace for new files, at least it did when I changed it to `Microsoft.Framework.Internal` just now)